packages/cli/src/utils.ts

```diff
@@ -114,19 +120,54 @@ export async function sleep(ms: number): Promise<void> {
 }
 
 export function getActiveEnv(): string {
-  return "default";
+  if (isMultiEnvEnabled()) {
```

Comment:
Can we use `environmentManager.getDefaultEnvName()` here?

Reply:
Yes. Fixed